products/container/api.yaml

```diff
@@ -348,15 +402,77 @@ objects:
                   the cluster. When enabled, it ensures that a Heapster pod is
                   running in the cluster, which is also used by the Cloud
                   Monitoring service.
-      # TODO(nelsonjr): This is a ResourceRef but on a resource on another
-      # product (Compute). Figure out how to represent/implement this. For now
-      # making it a string.
+          # kubernetesDashboard is deprecated.
+          - !ruby/object:Api::Type::NestedObject
+            name: 'networkPolicyConfig'
+            description: |
+              Configuration for NetworkPolicy. This only tracks whether the
+              addon is enabled or not on the Master, it does not track whether
+              network policy is enabled for the nodes.
+            properties:
+              - !ruby/object:Api::Type::Boolean
+                name: 'disabled'
+                description: Whether NetworkPolicy is enabled for this cluster.
       - !ruby/object:Api::Type::String
         name: 'subnetwork'
         description: |
           The name of the Google Compute Engine subnetwork to which the cluster
           is connected.
-      # 'enableKubernetesAlpha' not supported: we are only producing GA API.
+      # TODO: figure out nodePools[]
+      - !ruby/object:Api::Type::Array
+        name: 'locations'
```

Comment:
Sorry for the drive-by! The `gcloud` name for this field is _much_ more clear, `nodeLocations`. We've adopted it in Terraform as well (https://www.terraform.io/docs/providers/google/r/container_cluster.html#node_locations).

Would it be worth using this in Ansible / InSpec too?

Reply:
I can't speak to InSpec, but Ansible has a concept of aliasing, where one field has multiple names. I'll add an alias for `nodeLocations`.

Reply:
If/when we move TF to use MM for GKE is having different names going to be a problem? And do you enjoy acronyms as much as I do?

Reply:
MM overrides let us change names easily, so that's not a problem at all.